src/util/measurementUncertaintyTool.js

```diff
@@ -0,0 +1,66 @@
+import Decimal from 'decimal.js';
+
+// Run node version with 14.18.3
+/**
+ * Custom measurement uncertainty rounding rules.
+ * @param  {number} inputValue            measured value from annotation
+ * @param  {number} uncertaintyValue      uncertainty from pixel diagonal
+ */
+
+/**
+ * Rounding
+ */
+
+function roundValue(inputValue, uncertaintyValue) {
+  const value = new Decimal(inputValue);
+  let i = 0;
+
+  // Check if uncertainty is smaller than 1
+
+  if (uncertaintyValue < 1) {
+    const belowOne = uncertaintyValue.toString().split('.')[1];
+    /**
+      Uncertainty rounding measurement rule
+      regarding first significant figure:
+      if the first significant figure is greater than 2,
+      rounding digit stays in the same index.
+      However if it is 1 or 2, add one more 0 in the decimal.
+      **/
+
+    for (i = 0; i <= belowOne.length; i++) {
```

Comment:
I believe this loop would cause an error if i = belowOne.length
```suggestion
    for (i = 0; i < belowOne.length; i++) {
```